tests/rptest/tests/schema_registry_test.py

```diff
@@ -986,14 +1083,32 @@ def test_post_compatibility_subject_version(self):
             subject=f"{topic}-key", version=1, data=schema_2_data)
         assert result_raw.status_code == requests.codes.ok
         assert result_raw.json()["is_compatible"] == True
+        assert len(result_raw.json()["messages"]) == 0
 
-        self.logger.debug("Check compatibility backward, no default")
+        self.logger.debug("Check compatibility backward, no default, verbose")
         result_raw = self._post_compatibility_subject_version(
-            subject=f"{topic}-key", version=1, data=schema_3_data)
+            subject=f"{topic}-key",
+            version=1,
+            data=schema_3_data,
+            verbose=True)
         assert result_raw.status_code == requests.codes.ok
         assert result_raw.json()["is_compatible"] == False
 
+        self.logger.debug(
+            "Check compatibility backward, no default, not verbose")
```

Comment:
I know this is just a logger but I don't understand what it's supposed to mean...